2996_reflection/reflection.md

```diff
@@ -6337,17 +6342,20 @@ namespace std::meta {
 
   consteval bool has_default_member_initializer(info r);
 
-  consteval bool has_parent(info r);
-  consteval info parent_of(info r);
-
+  consteval bool $has-type$(info r); // exposition only
```

Comment:
I don't think we need this one in the synopsis. We don't do that for any other function...

Reply:
Daniel asked for it, and I think [he's probably right](https://www.google.com/search?q=synopsis+%22exposition+only%22+site%3Aeel.is&sca_esv=8ffc4cc3ad7d481a&rlz=1C5GCCM_en&ei=uC4JaPDNKaOs5NoPyJKQuAw&ved=0ahUKEwjwiabp4e6MAxUjFlkFHUgJBMcQ4dUDCBA&uact=5&oq=synopsis+%22exposition+only%22+site%3Aeel.is&gs_lp=Egxnd3Mtd2l6LXNlcnAiJnN5bm9wc2lzICJleHBvc2l0aW9uIG9ubHkiIHNpdGU6ZWVsLmlzSIAXUIwHWOwVcAF4AJABAJgBggKgAYwJqgEGMTguMC4xuAEDyAEA-AEBmAIAoAIAmAMAiAYBkgcAoAfXBrIHALgHAA&sclient=gws-wiz-serp). We probably ought to consider doing it for the other "exposition-only" functions, too.

Reply:
Good point; removed.

Reply:
e.g. [this one](https://eel.is/c++draft/ranges#range.utility.conv.general-5) isn't in the `<ranges>` synopsis.

Reply:
Done.